packages/react-reconciler/src/forks/ReactFiberErrorDialog.native.js

```diff
@@ -7,17 +7,32 @@
  * @flow
  */
 
-import type {CapturedError} from '../ReactCapturedValue';
+import type {Fiber} from '../ReactFiber';
+import type {CapturedValue} from '../ReactCapturedValue';
+
+import {ClassComponent} from '../ReactWorkTags';
 
 // Module provided by RN:
 import {ReactFiberErrorDialog as RNImpl} from 'react-native/Libraries/ReactPrivate/ReactNativePrivateInterface';
+
 import invariant from 'shared/invariant';
 
 invariant(
   typeof RNImpl.showErrorDialog === 'function',
   'Expected ReactFiberErrorDialog.showErrorDialog to be a function.',
 );
 
-export function showErrorDialog(capturedError: CapturedError): boolean {
+export function showErrorDialog(
+  boundary: Fiber,
+  errorInfo: CapturedValue<mixed>,
+): boolean {
+  const capturedError = {
+    componentStack: errorInfo.stack !== null ? errorInfo.stack : '',
+    error: errorInfo.value,
+    errorBoundary:
```

Comment:
I think you can drop this field. https://github.com/facebook/react-native/blob/21733641a8f1da6d6721618a0180509ab2b080a1/Libraries/Core/ReactFiberErrorDialog.js#L31

And maybe remove things from the type on their end.